code/modules/mob/living/carbon/human/species_types/android.dm

```diff
@@ -2,27 +2,50 @@
 	name = "Android"
 	id = "android"
 	say_mod = "states"
-	species_traits = list(NOBLOOD, NO_DNA_COPY, NOTRANSSTING)
-	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_GENELESS,TRAIT_NOFIRE,TRAIT_PIERCEIMMUNE,TRAIT_NOHUNGER,TRAIT_LIMBATTACHMENT)
+	species_traits = list(NOBLOOD, NOZOMBIE, NOHUSK, NO_DNA_COPY, NOTRANSSTING)
```

Comment:
```suggestion
	species_traits = list(NOBLOOD, NOZOMBIE, NOHUSK, NO_DNA_COPY, NOTRANSSTING, ROBOTIC_LIMBS)
```

Reply:
so they can't get fleshy limbs added via surgery.
You'll want to test this one just to make sure they don't turn invisible for some reason (rendering code is wonky)

Reply:
Adding that makes them turn invisible except for eyes, arms and legs.

Reply:
right, torso and head, the gendered parts